docs/usage/contrib/sqlalchemy/models_and_repository.rst

```diff
@@ -43,6 +43,7 @@ Both include a ``BigInteger`` based primary key
 and ``BigIntAuditBase`` includes an ``updated`` and ``created`` timestamp column.
 
 Models using these bases also include the following enhancements:
+
 * Auto-generated snake-case table name from class name
 * Pydantic BaseModel and Dict classes map to an optimized JSON type that is
 :class:`JSONB <sqlalchemy.dialects.postgresql.JSONB>` for the Postgres and
```

Comment:
```suggestion
   :class:`JSONB <sqlalchemy.dialects.postgresql.JSONB>` for the Postgres and
```

Line below needs fixing to but I can't make a suggestion for that :upside_down_face:

Reply:
I think the full fix to not have `bullet with no line ending` is
```
Models using these bases also include the following enhancements:

* Auto-generated snake-case table name from class name

* Pydantic BaseModel and Dict classes map to an optimized JSON type that is
  :class:`JSONB <sqlalchemy.dialects.postgresql.JSONB>` for the Postgres and
  :class:`JSON <sqlalchemy.types.JSON>` for other dialects.
```